doc/nrf/includes/apply_cmake_variable.txt

```diff
@@ -0,0 +1,36 @@
+.. |makevarformatstart| raw:: html
+
+   <strong class="makevar">
+
+.. |makevarformatend| raw:: html
+
+   </strong>
+
+.. |fileformatstart| raw:: html
+
+   <code class="file docutils literal notranslate"><span class="pre">
+
+.. |fileformatend| raw:: html
+
+   </span></code>
+
+To enable |variable_feature|, build the application with the |makevarformatstart|\ |makevar|\ |makevarformatend| variable set to |fileformatstart|\ |cmake_file_name|\ |fileformatend|.
```

Comment:
When I'm reading the document, I'm looking for a twist in the text saying that this is one example of building.

```suggestion
For example, to enable |variable_feature|, build the application with the |makevarformatstart|\ |makevar|\ |makevarformatend| variable set to |fileformatstart|\ |cmake_file_name|\ |fileformatend|.
```